modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2142,6 +2142,93 @@ definition {
 			viewHomeFolder = "Global");
 	}
 
+	@description = "LPS-156704 - Verify that the Includes operator is available when adding filters for the picklist fields to select data from the object"
+	@priority = "4"
+	test IncludesIsAvailableForPicklistFields {
+		task ("Add a Custom Picklist and Item (via API)") {
+			Picklist.addPicklistViaAPI(picklistName = "Picklist Test");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "PicklistItemTest",
+				itemName = "Picklist Item Test",
+				picklistName = "Picklist Test");
+		}
+
+		task ("Add a Custom Object and Custom Picklist (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161898",
+				objectName = "CustomObject161898",
+				pluralLabelName = "Custom Objects 161898");
```

Comment:
Use the ticket number correct to add the Custom Object - [LPS-165980](https://issues.liferay.com/browse/LPS-165980)